rules/windows/process_creation/proc_creation_win_hktl_powersploit_empire_default_schtasks.yml

```
@@ -29,17 +29,14 @@ detection:
         Image|endswith: '\schtasks.exe'
         CommandLine|contains|all:
             - '/Create'
-            - '/SC'
-            - '/TN'
-            - 'Updater'
-            - '/TR'
-            - 'powershell'
+            - 'powershell.exe -NonInteractive'
```

```suggestion
            - 'powershell.exe -NonI'
```
https://github.com/EmpireProject/Empire/blob/08cbd274bef78243d7a8ed6443b8364acd1fc48b/lib/modules/powershell/persistence/userland/schtasks.py#L219